pages/memgraph-lab/getting-started/installation-and-deployment.mdx

```diff
@@ -0,0 +1,350 @@
+
+Visit the [Download Hub](https://memgraph.com/download/#individual) and download
+the Memgraph Lab desktop application. You can install Memgraph Lab by
+double-clicking the downloaded installer and following the instructions.
+
+If you downloaded Memgraph Lab on Linux, execute:
+
+```
+sudo dpkg -i MemgraphLab-x.x.x.deb
+```
+
+<Callout type="info"> If you encounter a security warning while installing
+Memgraph Lab on **Windows**, click on **More info** and verify that Memgraph is
+listed as the Publisher by clicking on **Run anyway**. </Callout>
+
+{<h3 className="custom-header">Connect to Memgraph</h3>}
+
+Once you've started Memgraph and installed Memgraph Lab, you can
+[connect](/memgraph-lab/getting-started/connection-types) to your running Memgraph instance
+through Memgraph Lab.
+
+</Steps>
+
+## Install Memgraph Lab with Docker
+
+Running Memgraph Lab in a Docker container provides a convenient and isolated
+environment for managing graph data. Follow these steps to get started:
+
+<Steps>
+
+{<h3 className="custom-header">Run Memgraph</h3>}
+
+Memgraph Lab requires a [running Memgraph database instance](/getting-started).
+
+If you installed Memgraph using Docker, ensure that ports `7687`, used for
+instance connections (`-p 7687:7687`), and `7444`, used for logs (`-p
+7444:7444`), are exposed.
+
+{<h3 className="custom-header">Pull the Memgraph Lab Docker image</h3>}
+
+Before running Memgraph Lab, ensure that you have
+[Docker](https://www.docker.com/) installed. Then, pull the latest Memgraph Lab
+image using the following command:
+
+```
+docker pull memgraph/lab:latest
+```
+
+{<h3 className="custom-header">Start the container</h3>}
+
+To start Memgraph Lab in a Docker container, run:
+
+```
+docker run -d -p 3000:3000 --name lab memgraph/lab
+```
+
+Once the container is up you can access Memgraph Lab on
+`localhost:3000`.
+
+<Callout type="info">
+When running Memgraph Lab with Docker or Docker Compose, you can
+[configure it using the environment variables](/memgraph-lab/configuration).
+</Callout>
+
+{<h3 className="custom-header">Connect to Memgraph</h3>}
+
+Once you've started Memgraph and installed Memgraph Lab, you can
+[connect](/memgraph-lab/getting-started/connection-types) to your running Memgraph instance
+through Memgraph Lab.
+
+</Steps>
+
+## Custom installation settings
+
+When deploying Memgraph Lab, you might need to customize certain settings to
+better align with your environment or deployment requirements. This section
+covers how to configure Memgraph Lab with a [custom base
+path](#custom-base-path) and use [custom SSL
+certificates](#custom-ssl-certificate) for secure communication. These settings
+are especially useful when integrating Memgraph Lab with a reverse proxy or
+securing your application with SSL. 
+
+For further customization, refer to the
+[Configuration](/memgraph-lab/configuration) documentation to learn how to
+configure Memgraph Lab using environment variables.
+
+### Custom base path
```

Comment:
I am thinking if SSL and BASE_PATH should be subpages of the installation and deployment, especially now when we will be adding CONFIG for settings, node actions, datasets, etc.

Reply:
Created issue for the future improvements. https://github.com/memgraph/documentation/issues/1246